### PR TITLE
Update translation cell actions : add top level share action, and add reflect on dropdown for mobile

### PR DESCRIPTION
**Pull request by muhajirdev**
image

image

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -55,6 +56,7 @@ function MyApp({ Component, pageProps }): JSX.Element {
           </IdProvider>
         </ThemeProvider>
       </ReduxProvider>
+      <ToastContainer />
```
Comment (muhajirdev) [inline src/pages/_app.tsx:59]: don't worry about this for now 😁, I know we dont' want to always ToastContainer

File: src/components/dls/Toast/Toast.stories.tsx [hunk 2]
```diff
@@ -1,105 +1,105 @@
 /* eslint-disable react/no-multi-comp */
-import { ToastContainer, toast, ToastStatus } from './Toast';
-import styles from './Toast.stories.module.scss';
+// import { ToastStatus } from './Toast';
```
Comment (muhajirdev) [inline src/components/dls/Toast/Toast.stories.tsx:2]: Gonna fix and optimize the toast DLS in another PR to keep this PR small

File: locales/en/common.json [hunk 3]
```diff
@@ -156,6 +156,7 @@
         "translation": "Translation"
     },
     "reciter": "Reciter",
+    "reflect": "Reflect",
     "reflect-this-verse": "Reflect on this verse",
```
Comment (tifa2UP) [inline locales/en/common.json:160]: delete this

File: src/components/QuranReader/TranslationView/ShareVerseButton.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+type ShareVerseButtonProps = {
+  verseKey: string;
+};
+
+export const onShareClicked = (verseKey, callback: () => void) => {
+  const origin = getWindowOrigin();
+  const [chapter, verse] = getVerseAndChapterNumbersFromKey(verseKey);
+  if (origin) {
+    clipboardCopy(`${origin}/${chapter}/${verse}`).then(callback);
```
Comment (tifa2UP) [inline src/components/QuranReader/TranslationView/ShareVerseButton.tsx:19]: This is an example of a place where we should be doing the logging. You can sync with Osama on how it should be done

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**